components/server/src/workspace/config-provider.ts

```diff
@@ -310,3 +344,17 @@ export class ConfigProvider {
     }
 
 }
+
+export class InvalidGitpodYMLError extends Error {
```

Comment:
We have quite some classes like this around. Would be awesome if we found a way to generalize this approach! :+1: